### PR TITLE
Fixed #10052 - Icon support for TriStateCheckBox

### LINKED ISSUE #10052: Icon support for TriStateCheckBox
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[X] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**

Unable to change the classes for the True and False icons

**Expected behavior**

Ability to change the icons of the True and False states in the same way the standard Checkbox icon allows.

### PR DESCRIPTION
**Pull request by iuribezerra**
Added icons in the tristatcheckbox. Two parameters were created to define the correct icons. 
 - By default, strings (pi-check and pi-pi-times) remain.

### CODE REVIEW

**Comment by yigitfindikli** [comment]
Hi @iuribezerra,

Thanks for your contribution.

Regards.

File: src/app/components/tristatecheckbox/tristatecheckbox.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const TRISTATECHECKBOX_VALUE_ACCESSOR: any = {
             </div>
             <div class="p-checkbox-box" (click)="onClick($event,input)"  role="checkbox" [attr.aria-checked]="value === true"
                 [ngClass]="{'p-highlight':value!=null,'p-disabled':disabled,'p-focus':focused}">
-                <span class="p-checkbox-icon pi" [ngClass]="{'pi-check':value==true,'pi-times':value==false}"></span>
+                <span class="p-checkbox-icon" [ngClass]="value==true ? checkboxTrueIcon : checkboxFalseIcon"></span>
```
Comment (luehmann) [inline src/app/components/tristatecheckbox/tristatecheckbox.ts:20]: This also shows the checkBoxFalse icon in the null state

**Review by luehmann — COMMENTED** [review]
Bug where null state shows false icon

**Comment by luehmann** [comment]
See #10101

**Pull request merged**